### ISSUE TITLE
Settings modifications are not reflected in the activity bar view

### ISSUE THREAD

**Issue by Kasher** [fabiospampinato/vscode-todo-plus#251]
Hi,

It seems that if I perform changes to the extension's settings, they are not reflected to the activity bar view, but only to the actual todo files.

I changed the settings as follows:
```
    "todo.indentation": "    ",
    "todo.timekeeping.created.enabled": true,
    "todo.file.name": "TODO.TODO",
```

And I have to following `TODO.TODO` file:
![DeepinScreenshot_select-area_20200330234507](https://user-images.githubusercontent.com/5157107/77960199-8ce0b980-72e0-11ea-8bda-42a2f721c2c3.png)

As you can see, the activity bar view is completely empty, but the todo file actually uses 4 spaces as indentation and it is called `TODO.TODO`.

If I create a file called `TODO`, and use 2 spaces as indentation, I see the content of this file in the activity bar view (these are the default settings).

My VSCode version is 1.43.1, and the version of `TODO+` is 4.15.0.

Let me know if more info is needed.

Many thanks!

**Comment by fabiospampinato** [comment]
Did you reload the app?

**Comment by Kasher** [comment]
Yes, I tried to relaunch the app several times.
Thanks for the quick response @fabiospampinato !

**Comment by fabiospampinato** [comment]
Can you post your TODO.TODO file?

**Comment by Kasher** [comment]
That's what I used as its content:
```
Todo:
    Test
    Test 2
```

**Comment by fabiospampinato** [comment]
> I just thought about something - I'm using remoteSSH extension and loading a remote workspace. Any chance it is related?

Probably, yeah. Everything else seems fine. Could you try using just a regular vscode workspace with the same settings?

**Comment by Kasher** [comment]
Ok. So I tried using a local workspace, and still got the same result - unless I name my file `TODO` and use 2 spaces for indentation, nothing is shown in the activity bar.

What other info might be relevant?

**Comment by fabiospampinato** [comment]
🤔 I'm not sure, maybe the values provided in `todo.file.include` are case-sensitive, the fault ones are these: https://github.com/fabiospampinato/vscode-todo-plus/blob/adbd36d5b323110b5e680a6149b5f14b9da6fe11/package.json#L66-L74

Maybe you need to add `**/*.TODO` to that setting. If that works I should push an update of the extension with this fix.

**Comment by Kasher** [comment]
That did the trick! Nice catch!!!
Thanks so much :)

**Comment by fabiospampinato** [comment]
This should now be fixed in v4.16, without needing to change that setting.
